framework/test/mocha/functional/http/get/delegates.js

```diff
@@ -640,6 +640,17 @@ describe('GET /delegates', () => {
 			});
 		});
 
+		it('using limit=101 should sort forgers ascending using nextSlot', async () => {
+			return forgersEndpoint.makeRequest({ limit: 101 }, 200).then(res => {
+				const forgers = _.clone(res.body.data);
+				expect(
+					forgers
+						.reverse()
+						.sort((prev, next) => (prev.nextSlot > next.nextSlot ? 1 : -1))
```

Comment:
this doesn't seem to test what it should test.
If this logic is wrong, source and test will still say it is correct

Reply:
Updated test to make it independent